cobbler.spec

```diff
@@ -1,6 +1,17 @@
+#
+# spec file for package cobbler
+#
+# Copyright (c) 2021 SUSE LLC
```

Comment:
Please add the copyright line for Red Hat.

Reply:
Hm adding additional copyrights for other persons/companies is somewhat problematic.
I found a submit from 2018, next from 2015 with a @redhat.com address as author.

I know that the tool is initially coming from Redhat, from Michael DeHaan afaik, and I kept this of course:

# Copyright (c) 2006 Michael DeHaan <mdehaan@redhat.com>

adding a:
# Copyright (c) 2018 RedHat
# Copyright (c) 2015 RedHat
does not make sense.

# Copyright (c) 2006 RedHat

would make sense if this is the full company string typically used, but it looks wrong that I am adding this.

While SUSE was the driver for this project, at least for 2020 and 2021, I'd try to remove this one:
# Copyright (c) 2021 SUSE LLC

which came from open build service auto-license  creation.

Hmm, afaik you are working for RedHat as well? As you did actively help and contribute quite a lot..., but then please tell me the exact line I should add:

# Copyright (c) 2021 RedHat

I'll add this..., would be great if you are ok with it...
This is really not about honours, company, whatever matters.. It's simply to feed the build service and to be able to move on...

Reply:
The reason the SUSE copyright line gets added is that the spec file formatter blindly adds it. It should not, and I wish they'd stop doing that, but it does now. 

The Red Hat copyright would be something like:

```
# Copyright (c) 2006-2012 Red Hat, Inc.
```

My copyright on the spec file would be:

```
# Copyright (c) 2019 Neal Gompa <ngompa13@gmail.com>
```

I don't work for Red Hat. 😉